src/Symfony/Component/Finder/Comparator/Comparator.php

```diff
@@ -36,23 +36,9 @@ public function __construct(string $target = null, string $operator = '==')
      */
     public function getTarget()
```

Comment:
adding the return type is for another iteration?

Reply:
Yes please. 🙂